### PR TITLE
Phillip Galbraith Updated Styles, added Links

### PR DESCRIPTION
**Pull request by phillipgalbraith**
Fixed the Global Styles and added more styled components: CreatePotluckPage, Signup
Added link at Login to Signup, and link at Dashboard to CreatePotluckPage

### CODE REVIEW

File: potluck/src/components/CreatePotluckPage.js [hunk 1]
```diff
@@ -29,28 +45,30 @@ const CreatePotluckPage = () => {
     });
   };
   return (
-    <div>
```
Comment (phillipgalbraith) [inline potluck/src/components/CreatePotluckPage.js:32]: Added styled component tag and indented.

File: potluck/src/components/Dashboard.js [hunk 2]
```diff
@@ -83,8 +105,8 @@ const Dashboard = () => {
   }, [confirmed]);
   
   const confirmClick = (e) => {
-      e.preventDefault();
-      setConfirmed(!confirmed);
```
Comment (phillipgalbraith) [inline potluck/src/components/Dashboard.js:87]: corrected indentation

File: potluck/src/components/Dashboard.js [hunk 3]
```diff
@@ -104,7 +126,12 @@ const Dashboard = () => {
   return (
     <StyledDashboard>
       <h1 className="pageTitle">Dashboard</h1>
-      <button className="styledButton" onClick={newPotluck}>Create New Potluck</button>
```
Comment (phillipgalbraith) [inline potluck/src/components/Dashboard.js:107]: replaced button with  a <Link> wrapper on a button and Route

File: potluck/src/components/Signup.js [hunk 4]
```diff
@@ -24,59 +31,61 @@ const Signup = () => {
   }
 
   return (
-  <section>
```
Comment (phillipgalbraith) [inline potluck/src/components/Signup.js:27]: Changed indentation after adding Parent tag <StyledSignUp>

**Review by ghostman1996 — APPROVED** [review]
All looks good.

**Pull request merged**